dags/dag.py

```diff
@@ -1,22 +1,22 @@
 from datetime import datetime
-import os
-import json
 from airflow import DAG
 from airflow.operators.bash_operator import BashOperator
 from airflow.operators.python_operator import PythonOperator
-from azure.keyvault.secrets import SecretClient
-from azure.identity import ManagedIdentityCredential
+from airflow.models import Variable
 
-client_id = os.getenv("USER_ASSIGNED_MANAGED_IDENTITY")
-credential = ManagedIdentityCredential(client_id=client_id)
+try:
+    username = Variable.get("DecosUsername")
+    password = Variable.get("DecosPassword")
+    url = Variable.get("DecosURL")
+except Exception as e:
```

Comment:
Does this exception trigger btw? Because if you try/catch everything w/o reraising, your code will exit with code 0, but it might fail somewhere without you knowing

Reply:
you might want to consider removing the try/except stuff, so you know if and when something fails

Reply:
Nee, dit is idd een try except die ik misschien wel moet raise

Reply:
De rest negeer ik, zodat ik verschillende pods kan testen